week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,20 @@
+fetch("https://xkcd.now.sh/?comic=latest")
+  .then((response) => {
+    if (response.status >= 200 && response.status <= 299) {
```

Comment:
this is much better, but there is no 299 code, I will say between 200 and 304 makes more sense